index.html

```diff
@@ -620,12 +684,15 @@ <h3>Validate Algorithm</h3>
 <a href="#bitstring-expansion-algorithm">Bitstring Expansion Algorithm</a>.
         </li>
         <li>
-Let <strong>status</strong> be the value of the bit at position
-<strong>credentialIndex</strong> in the <strong>revocation bitstring</strong>.
+Let <strong>status</strong> be the value at position indicated by the
+<strong>credentialIndex</strong> times the <strong>size</strong> in the 
+<strong>revocation bitstring</strong>.
         </li>
         <li>
-Return <code>true</code> if <strong>status</strong> is 1, <code>false</code>
-otherwise.
+For <code>statusPurpose</code> of <code>revocation</code> or <code>suspension</code> 
+return <code>true</code> if <strong>status</strong> is 1, <code>false</code>
+otherwise.  Otherwise, return the corresponding <code>value</code> of the status
+as indicated in the <code>message</code> array.
```

Comment:
The use of "message" is confusing me -- perhaps an example would help. The "message" field sounds more like "statusEnum" or something like that? Are we trying to model an enumerated type, essentially? And "status" becomes a value that can be used to look up the enumerated type name? This seems confusing as well ... because we're calling the value of the position in the bistring `status` and we've got a `statusPurpose` of `status`. I think `enumeration` or `message` (or something else) is a better name for the latter. We definitely shouldn't reuse `status`, IMO.

Reply:
very much agree with this, I think it should be called message.